test/cypress/integration/field-extension.spec.js

```diff
@@ -0,0 +1,27 @@
+import { entry } from '../utils/paths'
+
+const post = {
+  id: '1MDrvtuLDk0PcxS5nCkugC',
+  title: 'My first post'
+}
+
+context('Field extension', () => {
+  beforeEach(() => {
+    cy.setAuthTokenToLocalStorage()
+  })
+
+  it('opens first post and checks that field extension is rendered', () => {
+    cy.visit(entry(post.id))
+
+    cy.getByText(post.title).should('exist')
+
+    // eslint-disable-next-line
+    cy.wait(3000)
```

Comment:
can we listen to `onload` of the frame instead of hardcoding reasonable but arbitrary number?

Reply:
Unfortunately `onload` doesn't work was expected in Cypress and it's the only working solution I could find.

Reply:
Let's keep it as it is right now but I'd also research extending the Web App so it can notified us "the channel is ready"

Reply:
Code like that may lead to flakyiness in the future.